packages/p2p/stores/buy-sell-store.js

```diff
@@ -84,9 +85,12 @@ export default class BuySellStore {
             p2p_advertiser_info: 1,
         }).then(response => {
             if (!response.error) {
-                const { p2p_advertiser_info } = response;
-                this.setContactInfo(p2p_advertiser_info.contact_info);
-                this.setPaymentInfo(p2p_advertiser_info.payment_info);
+                const { contact_info, payment_info } = response.p2p_advertiser_info;
+                this.setContactInfo(contact_info);
+                this.setPaymentInfo(payment_info);
+                if(contact_info && payment_info){
+                    this.setHasProfileInfo(true)
+                }
```

Comment:
We can make `has_payment_info` a `computed`.

Reply:
Carol, this observable only used once on initalErrors and I think we don't need to check it each time that contact_info and payment_info change because the validation function check them